res/css/views/dialogs/_InviteDialog.scss

```diff
@@ -256,10 +256,6 @@ limitations under the License.
 
 .mx_InviteDialog_helpText {
     margin: 0;
-
-    .mx_AccessibleButton_kind_link {
-        padding: 0;
-    }
```

Comment:
`AccessibleButton` with `kind=link` is not used here.

![after3](https://user-images.githubusercontent.com/3362943/174980100-6cbb7bb3-3f31-4a69-8f34-50837d455229.png)